DotBot.py

```diff
@@ -25,8 +25,9 @@ def __init__(self, dotBotId, x, y, floorplan):
         # singletons
         self.simEngine            = SimEngine.SimEngine()
         self.wireless             = Wireless.Wireless()
-        # last received seqNumMovement (to filter out duplicate commands)
+        # last received seqNumMovement (to filter out duplicate commands and notifications)
```

Comment:
change to `# sequence numbers (to filter out duplicate commands and notifications)`